packages/lean-imt/src/lean-imt.ts

```diff
@@ -330,17 +330,31 @@ export default class LeanIMT<N = bigint> {
      * It imports an entire tree by initializing the nodes without calculating
      * any hashes. Note that it is crucial to ensure the integrity of the tree
      * before or after importing it.
-     * The tree must be empty before importing.
+     * @param hash The hash function used to create nodes.
      * @param nodes The stringified JSON of the tree.
+     * @param reviver A function that transforms the results. This function is called for each node of the tree and can
+     * be used to convert the node types.
+     * @returns A LeanIMT instance.
      */
-    public import(nodes: string) {
+    static import<N = string>(
+        hash: LeanIMTHashFunction<N>,
+        nodes: string,
+        reviver?: (this: any, key: string, value: any) => any
```

Comment:
There's an asymmetry that export() automatically handles conversion of bigints to strings, while import() requires the user to explicitly write the code to do that.  Seems like there should be a default behavior so that export() followed by import() can work in a default way.  Put a different way, I'd expect when using bigint that I'd either have to specify a conversion for both import and export, or for both, but not for only one of them.

Reply:
True, N is `bigint` by default in the `import` function too now. And there's a conversion from strings to bigints if the reviver function is not defined.